docs/docs/20-usage/51-plugins/10-plugins.md

```diff
@@ -1,6 +1,8 @@
 # Plugins
 
-Plugins are Docker containers that perform pre-defined tasks and are configured as steps in your pipeline. Plugins can be used to deploy code, publish artifacts, send notification, and more.
+Plugins are pipeline steps that perform pre-defined tasks and are configured as steps in your pipeline. Plugins can be used to deploy code, publish artifacts, send notification, and more.
+
+They are automatically pulled from [plugins.drone.io](https://plugins.drone.io).
```

Comment:
Domain is http only:

```suggestion
They are automatically pulled from [plugins.drone.io](http://plugins.drone.io).
```

Is it really taking the plugins from that page or is it just an index?

Reply:
wait it's http ?!!

Reply:
Yip 😢 Maybe we should start some own marktplace with an repo anyone can share their own plugins with the community by some PR.

Reply:
yes that's a good idear! this doesnt look well maintained anymore :/

Reply:
I noticed that and hoped that it was just an issue on my side... It's just an index, it takes the actual images from hub.docker.com. Example: <https://hub.docker.com/r/plugins/docker>

Reply:
Crazy they really claimed `plugins` org at docker-hub. Searched for days to find the image resolution step 🤦🏾

Reply:
Damn, we should probably fork `plugins/git` (runs in every pipeline!) as it's unmaintained... https://github.com/drone-plugins/drone-git

Reply:
> Crazy they really claimed `plugins` org at docker-hub. Searched for days to find the image resolution step 🤦🏾

Same lol

Reply:
see #303